package.json

```diff
@@ -33,13 +33,12 @@
     "@salesforce/core": "^2.0.1",
     "@salesforce/telemetry": "^1.2.2",
     "@salesforce/ts-types": "^1.1.4",
-    "@types/shelljs": "^0.8.8",
     "ajv": "^6.12.4",
     "chalk": "^4.0.0",
     "cli-ux": "^5.4.5",
     "colors": "^1.4.0",
-    "listr": "^0.14.3",
-    "shelljs": "^0.8.3",
+    "enquirer": "^2.3.6",
+    "listr2": "^3.3.1",
```

Comment:
I got rid of `shelljs` since we are not using it (a long time ago we were, but not anymore). Also instead of `listr` I'm going to use `listr2` (and `enquirer` which `listr2` needs).

`listr` is a nice library that we can use to print command progress but is seems to be dead, as it has not taken fixes/updates in a long time. `listr2` is an effort by someone else to update `listr` and add more features to it.

In fact, there is [a thread](https://github.com/SamVerschueren/listr/issues/143) where the creator of `listr` and the creator of `listr2` and few others are discussing the idea of merging all of these back into one library at some point.

But until then, we should use `listr2` b/c it has the same MIT license, has all of the `listr` features, has many more bug fixes, and has more features that are useful to us (such as being able to use Promises as tasks).